editor/src/EditorStartup.cpp

```diff
@@ -46,29 +46,26 @@ void EditorStartup::initializeDefaultInputs() const
 }
 
 EditorStartup::EditorStartup()
-    : m_fixedUpdate{[&](double fixedUnscaledDeltaTime, double fixedDeltaTime)
-      {
+    : m_fixedUpdate{[&](double fixedUnscaledDeltaTime, double fixedDeltaTime) {
           m_engine->physXSystem.advance(fixedDeltaTime);
       }},
       m_update{[&](double unscaledDeltaTime, double deltaTime) {
           m_engine->sceneManager.update();
           m_engine->inputManager.processInput();
+          m_engine->soundSystem.update();
```

Comment:
There never is a situation when we can play sound in the editor without playing the game.
I believe it is unnecessary to update the sound system here.

```suggestion
```

Reply:
It's not about playing sound but updating settings within the editor.